Project_1.rb

```diff
@@ -0,0 +1,109 @@
+# definition of methods
+def add(num1,num2)
+  return num1 + num2
+end
+
+def subtract(num1,num2)
+  return num1 - num2
+end
+
+def multiply(num1,num2)
+  return num1 * num2
+end
+
+def divide(num1,num2)
+  return num1 / num2
+end
+
+def numberic?(string)
+  /[%\+\*\d]/.match(string)
+end
+
+def empty?(input)
+  if input.empty?
+    puts "You did not enter anything!"
+  end
+end
+
+def type?(input)
+  if /\A\d+\z/.match(input)
+    return "integer"
+  elsif /\A\d+\.\d+\z/.match(input)
```

Comment:
I really like that you've split all these off into helper methods - good organization!